test/parallel/test-sync-io-option.js

```diff
@@ -20,7 +20,7 @@ if (process.argv[2] === 'child') {
     execFile(process.execPath, args, function(err, stdout, stderr) {
       assert.equal(err, null);
       assert.equal(stdout, '');
-      if (/WARNING[\s\S]*fs\.readFileSync/.test(stderr))
+      if (/WARNING[\s\S]*readFileSync/.test(stderr))
```

Comment:
@targos I think v8 doesn't provide a way to get the actually bound object name. Is there any APIs which we can leverage? Also, `PrintSyncTrace` has only references to the Stack Frames. They just give only the debug function names.